modules/tools/MB_GraphGenerator/README.md

```diff
@@ -0,0 +1,20 @@
+# MB_GraphGen
```

Comment:
folder name MB_GraphGenerator is wrong may be graph_generator or may be more helpful name?